entrypoint.sh

```diff
@@ -13,7 +13,14 @@ if [ ! -d "$HOME/.config/gcloud" ]; then
         exit 1
     fi
 
-    echo "$APPLICATION_CREDENTIALS" | base64 -d > /tmp/account.json
+    # Check if $APPLICATION_CREDENTIALS is a valid base64 encoded string or not
+    if [ "$APPLICATION_CREDENTIALS" = "$(echo $APPLICATION_CREDENTIALS | base64 --decode | base64)" ]; then
```

Comment:
```suggestion
    if [ "$APPLICATION_CREDENTIALS" = "$(echo "$APPLICATION_CREDENTIALS" | base64 --decode 2>/dev/null | base64)" ]; then
```

Reply:
Also Linux `base64` (but not the OSX one) errors when given invalid input so this could be:

```suggestion
    if (echo "$APPLICATION_CREDENTIALS" | base64 --decode >/dev/null 2>&1)") ; then
```